build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/ProcessLogcatTiming.cs

```diff
@@ -91,8 +97,11 @@ public override bool Execute ()
 					Log.LogMessage (MessageImportance.Normal, $"Last timing message: {(last - start).TotalMilliseconds}ms");
 
 					WriteResults ();
-				} else
-					Log.LogWarning ("Wasn't able to collect the performance data");
+				} else {
+					Log.LogError ("Application start wasn't detected. Could not collect performance data.");
+					reader.Close ();
```

Comment:
Can we put this `reader` in a `using` block instead? There might be other places we forget to close it when it has a `return` early.

Reply:
A [using block](https://github.com/xamarin/xamarin-android/blob/671f53f54d410bc038ce349b05b1e4c460c9a939/build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/ProcessLogcatTiming.cs#L26) is in play already, I was just following convention with the other `Close` call at the end of the file parse. I suppose these additions could be removed for cleanliness?

Reply:
Yeah we don’t need the `Close` calls at all here, then.